### ISSUE TITLE
problem with some characters

### ISSUE THREAD

**Issue by mecaz** [dhrone/pydPiper#82]
Hello, 

First of all thanks for your amazing project. 
I am using it with volumio and 128x64 OLED 
display type:  luma_i2c
I2C port: 1
I2C address: 0x3c
type of display: sh1106
width: 128
height: 64
pagefile: pages_ssd1306.py

everything ok but my display cant show theese characters: ğ ş ı Ğ Ş 
is there anyway to fix it? 
can we replace theese like below 
ğ with g, ş with s, ı with i ?

thanks

**Comment by dhrone** [comment]
Yes, but with some effort.  You would need to edit the font files for the fonts that you require those specific glyphs.  Designing glyphs for very low resolution fonts like the latin1_5x8 is also not the easiest activity.  If you are looking to add glyphs to one of the larger fonts like the Vintl01_10x16 it will be much easier.
